nikola/plugins/command/import_wordpress.py

```diff
@@ -49,10 +52,27 @@
 from nikola.plugins.basic_import import ImportMixin, links
 from nikola.nikola import DEFAULT_TRANSLATIONS_PATTERN
 from nikola.plugins.command.init import SAMPLE_CONF, prepare_config, format_default_translations_config
+from nikola.plugins.command import plugin
 
 LOGGER = utils.get_logger('import_wordpress', utils.STDERR_HANDLER)
 
 
+def install_plugin(site, plugin_name, output_dir=None, show_install_notes=False):
+    LOGGER.notice("Installing plugin '{0}'".format(plugin_name))
+    plugin_installer = plugin.CommandPlugin()
+    plugin_installer.set_site(site)
```

Comment:
We use our plugins as singletons.  Take the instance from `plugin_installer = site.plugin_manager.getPluginByName('plugin', 'Command').plugin_object`